plugins/social_network/social_network.cpp

```diff
@@ -27,13 +37,51 @@
 #  define DEFAULT_VOTE_LIMIT 10000
 #endif
 
+
+// Depth of comment_content information storage history.
+struct content_depth_params {
+    content_depth_params() {
+    }
+
+    inline bool miss_content() const {
+        return has_comment_title_depth && !comment_title_depth &&
+               has_comment_body_depth && !comment_body_depth &&
+               has_comment_json_metadata_depth && !comment_json_metadata_depth;
```

Comment:
Need additional checks on `delta_time`

Reply:
Why? This method describes case when you want not to store content at all.